bootstrap/templates/kubernetes/apps/tools/system-upgrade-controller/app/rbac.yaml.j2

```diff
@@ -11,3 +11,13 @@ subjects:
   - kind: ServiceAccount
     name: system-upgrade
     namespace: tools
+#% if bootstrap_distribution in ['talos'] %#
+---
```

Comment:
You should also wrap [this](https://github.com/onedr0p/flux-cluster-template/blob/main/bootstrap/templates/kubernetes/talos/talconfig.yaml.j2#L159-L168) around an if block because the ServiceAccount is only needed if they are using suc.

Reply:
It won't hurt leaving it on by default.